client/components/sidebar-navigation/index.jsx

```diff
@@ -31,11 +32,12 @@ class SidebarNavigation extends React.Component {
 	}
 };
 
-SidebarNavigation.propTypes = {
+SidebarNavigationSection.propTypes = {
 	title: PropTypes.string,
 	linkClassName: PropTypes.string,
 	sectionTitle: PropTypes.string,
-	sectionName: PropTypes.string.isRequired
+	sectionName: PropTypes.string.isRequired,
+	setLayoutFocus: PropTypes.func.isRequired,
 };
 
-export default SidebarNavigation;
+export default connect( null, { setLayoutFocus } )( SidebarNavigationSection );
```

Comment:
I don't know that it'll be a problem, but keep in mind that `connect` makes components pure by default, which may or may not cause regressions if the component wasn't written well enough to be pure.

https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options

If there's a concern, you can pass `{ pure: false }` as the fourth argument to `connect`.